docs/src/components/IconList/pojo.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Anchor exports icons at the same level as every other component so there's no way to differentiate
+them from another component. Rather than modifying anchor to make my life easier, I'm treating this
+as how any user might interact with the library, so I'm grabbing all the icons in this file and exporting
+them in order to loop over them all.
```

Comment:
Does `import * as Icons` not allow you to iterate through that object to perform this same operation?

Reply:
I'll have to ask you IRL for clarification on this because that's what this file is for, pulling in all the icons and doing an export so it can be pulled in as `import * as Icons`.